README.md

```diff
@@ -1,6 +1,159 @@
-# Regolamento Fantainformatica
-
-### Amministratori
+# Regolamento Fantainformatica 2021-22
 
 - Presidente: Andrea `@andrea9lli`
-- Vice-presidente: Marco `@mlsmrc`
\ No newline at end of file
+- Vice-presidente: Marco `@mlsmrc`
+
+## Modalità CLASSIC
+
+### Montepremi
+* Quota di partecipazione: **50€**
+* Campionato
+	* 1° posto: **200€** (50% del totale)
+	* 2° posto: **100€** (25% del totale)
+	* 3° posto: **50€** (12.5% del totale)
+* Coppa:
+	* Vincitore: **50€** (12.5% del totale)
+
+
+### Regole mercato iniziale
+* Partecipanti: **8** o superiori
+* Budget iniziale: **500** crediti
+* Base d'asta: **1**
+* Modalità d'asta: Asta a chiamata libera per ruolo (P -> D -> C -> A)
+* Una sessione di asta al termine della sessione di calciomercato estivo della Serie A (di solito 31 agosto). La data è da concordare
+* Verrà tenuta aperta una sessione di mercato solo scambi tra utenti fino all'inizio della successiva giornata
+
+### Regole mercato di riparazione
+* Una sola sessione di asta di riparazione al termine del calciomercato della Serie A (di solito 31 gennaio). La data è da concordare
+* Sessione mercato solo scambi attivo all'interno dell'app dal 1 gennaio
+* Verrà effettuata una riassegnazione dei crediti in ordine crescente e in base alla posizione in classifica dei giocatori al termine del girone di andata, così come segue:
+	* 1° posto: **100**
+	* 2° posto: **110**
+	* 3° posto: **120**
+	* 4° posto: **130**
+	* ...
+	* 8° posto: **170**
+
+* Numero **illimitato** di svincoli e acquisti
+* In caso di svincoli, l'utente riprenderà il:
+	* **100%** dei crediti in caso il giocatore vada in un altro campionato
+	* **50% -1**   dei crediti altrimenti (tranne per i giocatori pagati **1** credito, per i quali verrà restituito **1**)
+* Per avere diritto a recuperare i crediti, gli svincoli vanno dichiarati prima dell'inizio dell'asta di riparazione. Resterà comunque possibile partecipare ad aste per altri svincolati, pur non avendo slot disponibili, utilizzando eventuali crediti rimanenti e senza poter recuperare ulteriori crediti 
+* Nessuna possibilità di sostituire un giocatore in caso di infortunio lungo se non nell'asta del mercato di riparazione
+* Si può riacquistare un giocatore precedentemente svincolato pagandolo la stessa cifra che era stato pagato originariamente, nel caso non ci siano offerte di altri utenti
+* Non si può svincolare un giocatore acquistato nella stessa sessione del mercato di riparazione
+
+#### Valorizzazione scouting
+* L'acquisto di un calciatore durante l'asta estiva dà diritto alla possibilità di acquisto dello stesso (qualora continuasse a giocare nel campionato di Serie A), durante l'asta  estiva della stagione successiva pagandolo **il massimo fra il triplo della valutazione di fantacalcio.it e il triplo della spesa effettuata l'anno precedente** per acquistarlo (esempio: Vlahovic comprato a 5 ma valutato 20, può essere ricomprato a 60 in quanto il massimo fra 20x3 e 5x3)
```

Comment:
In caso subentrasse un nuovo membro nel Fantacalcio al posto di un'altra squadra, questo erediterebbe anche lo scouting (quindi la possibilità di comprare i giocatori) della vecchia squadra?

Reply:
Discutiamo di questo punto durante la prossima asta.
Per ora validiamo il regolamento.
Oppure una volta validato il regolamento apri una pull request con il cambio e lo discutiamo